examples/notify.py

```diff
@@ -18,7 +19,11 @@ async def listen(conn):
     async with conn.cursor() as cur:
         await cur.execute("LISTEN channel")
         while True:
-            msg = await conn.notifies.get()
+            try:
+                msg = await conn.notifies.get()
+            except psycopg2.Error as ex:
+                print("ERROR: ", ex)
+                return
```

Comment:
the `try/except` here could be replaced with a `with contextlib.suppress(psycopg2.Error)` (and the `print` dropped)